src/benchq/problem_embeddings/_qsp.py

```diff
@@ -151,22 +151,11 @@ def _replace_named_qubits(ops: Iterable[cirq.Operation]) -> List[cirq.Operation]
     ]
 
 
-XPOW_IDENTITY_1 = cirq.XPowGate(exponent=-1)
-XPOW_IDENTITY_2 = cirq.XPowGate(global_shift=-0.25)
+XPOW_X_1 = cirq.XPowGate(exponent=-1)
+XPOW_X_2 = cirq.XPowGate(global_shift=-0.25)
```

Comment:
Not sure what this is doing here? Do we only want to detect XPowGates with this particular gobal phase?